TruePath/AbsoluteDirectory.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 TruePath contributors <https://github.com/ForNeVeR/TruePath>
+//
+// SPDX-License-Identifier: MIT
+
+namespace TruePath;
+
+public static class AbsoluteDirectory
```

Comment:
First of all, I propose getting rid of separation to `AbsoluteDirectory` and `AbsoluteFile`, and move all these methods to a `public static class FileIO`.